db/fixtures/ae_datastore/ManageIQ/Deployment/ContainerProvider/System/StateMachine/Deployment.class/__methods__/provision.rb

```diff
@@ -1,34 +1,19 @@
-def get_custom_tag(type)
-  type + "_#{$evm.root['automation_task'][:id]}"
-end
-
 def create_custom_tag(name)
   tag_name = "#{name}_#{$evm.root['automation_task'][:id]}"
-  unless $evm.execute('tag_exists?', 'miq_openstack_deploy', tag_name)
+  unless $evm.execute('tag_exists?', 'miq_openshift_deploy', tag_name)
     $evm.log(:info, "********************** creating tag ***************************")
     $evm.execute('tag_create',
-                 'miq_openstack_deploy',
+                 'miq_openshift_deploy',
                  :name        => tag_name,
                  :description => tag_name)
   end
 end
 
-def tagged_tasks
-  tasks = []
-  $evm.vmdb('miq_provision_request').all.each do |prov|
-    next unless prov.get_tags[:miq_openstack_deploy] &&
-      (prov.get_tags[:miq_openstack_deploy].include?(get_custom_tag("master")) ||
-        prov.get_tags[:miq_openstack_deploy].include?(get_custom_tag("node")))
-    tasks << prov
-  end
-  tasks
-end
-
 def create_provision_requests(vm_base_name, num_of_vms, args, type)
   (1..num_of_vms).each do |num|
     args[2]['vm_name'] = vm_base_name + "_" + num.to_s
     request            = $evm.execute('create_provision_request', *args)
-    request.add_tag("miq_openstack_deploy", type + '_' + $evm.root['automation_task'][:id].to_s)
+    request.add_tag("miq_openshift_deploy", type + '_' + $evm.root['automation_task'][:id].to_s)
   end
```

Comment:
Better to collect the requests and add the list of IDs to the automation task by type.  Then these ID lists can be used in the `provision_tagged_tasks` method in PR #9230 to load the tasks.

```ruby
def create_provision_requests(vm_base_name, num_of_vms, args, type)
  (1..num_of_vms).collect do |num|
    args[2]['vm_name'] = vm_base_name + "_" + num.to_s
    $evm.execute('create_provision_request', *args)
  end
end
```

This will return the new requests to `provision_request` where you will store the IDs.